src/Cooked/MockChain/GenerateTx.hs

```diff
@@ -226,15 +217,48 @@ txOutRefsToTxInsReference =
     )
     . mapM Ledger.toCardanoTxIn
 
--- Convert a list of 'Api.TxOutRef' into a 'Cardano.TxInsCollateral'
-txOutRefsToTxSkelInsCollateral :: [Api.TxOutRef] -> TxGen (Cardano.TxInsCollateral Cardano.ConwayEra)
-txOutRefsToTxSkelInsCollateral =
-  throwOnToCardanoError "txOutRefsToTxInCollateral"
-    . fmap toTxInsCollateral
-    . mapM Ledger.toCardanoTxIn
-  where
-    toTxInsCollateral [] = Cardano.TxInsCollateralNone
-    toTxInsCollateral ins = Cardano.TxInsCollateral Cardano.AlonzoEraOnwardsConway ins
+-- Computes the collateral triplet from the fee and the collateral inputs in
+-- the context
```

Comment:
I have updated the comment accordingly:
```
-- | Computes the collateral triplet from the fees and the collateral inputs in
-- the context. What we call a collateral triplet is composed of:
-- * The set of collateral inputs
-- * The total collateral paid by the transaction in case of phase 2 failure
-- * An output returning excess collateral value when collaterals are used
-- These quantity should satisfy the equation (in terms of their values):
-- collateral inputs = total collateral + return collateral
```